### PR TITLE
changed final purity above 1 to equal 1 and fix some bugs in purity e…

### PR DESCRIPTION
**Pull request by kristenadaley**
…stimation

# Pull Request Checklists

**Important:** When opening a pull request, keep only the applicable checklist and delete all other sections.

### Required

- [x] I tested the new code for my use case (please provide a reproducible example of how you tested the new functionality)

- [ ] I ensured all dplyr functions that commonly conflict with other packages are fully qualified. 

This can be checked and addressed by running `check_functions.pl` and responding to the prompts. Test your code _after_ you do this.

- [ ] I generated the documentation and checked for errors relating to the new function (e.g. `devtools::document()`) and added `NAMESPACE` and all other modified files in the root directory and under `man`. 

### Optional but preferred with PRs

- [ ] I updated and/or successfully knitted a vignette that relies on the modified code (which ones?)

### Required

- [ ] I documented my function using [ROxygen style](https://jozef.io/r102-addin-roxytags/#:~:text=Inserting%20a%20skeleton%20%2D%20Do%20this,Shift%2BAlt%2BR%20).)

- [ ] All parameters for the function are described in the documentation and the function has a decriptive title. 

Example:
```
#' Use GISTIC2.0 scores output to reproduce maftools::chromoplot with more flexibility
#'
#' @param scores output file scores.gistic from the run of GISTIC2.0
#' @param genes_to_label optional. Provide a data frame of genes to label (if mutated). The first 3 columns must contain chromosome, start, and end coordinates. Another required column must contain gene names and be named `gene`. (truncated for example)
#' @param cutoff optional. Used to determine which regions to color as aberrant. Must be float in the range [0-1]. (truncated for example)
```

- [ ] My function uses a library that isn't already a dependency of GAMBLR and I made the package aware of this dependency using the function documentation `import` statment. 

Example:
```
#' @return nothing
#' @export
#' @import tidyverse ggrepel
```

## Checklist for changes to existing code

- [x] I added/removed arguments to a function and updated documentation for all changed/new arguments

- [x] I tested the new code for compatability with existing functionality in the Master branch (please provide a reprex of how you tested the original functionality)

### CODE REVIEW

File: R/utilities.R [hunk 1]
```diff
@@ -1157,10 +1157,11 @@ estimate_purity = function(in_maf,
   # Calculate purity, and if the number is larger 1 (100%) use the VAF calue instead
```
Comment (mattssca) [inline R/utilities.R:1157]: Found a tiny typo, "VAF value", not "calue". But this was something previous.

File: R/utilities.R [hunk 2]
```diff
@@ -1113,7 +1113,7 @@ estimate_purity = function(in_maf,
   }else{
     # If no seg file was provided and assume_diploid paramtere is set to true,
     if(assume_diploid){
-      CN_new <- assign_cn_to_ssm(maf_file = in_maf, assume_diploid = TRUE,coding_only=coding_only,genes=genes)
+      CN_new <- assign_cn_to_ssm(maf_file = in_maf, assume_diploid = TRUE,coding_only=coding_only,genes=genes)$maf
```
Comment (mattssca) [inline R/utilities.R:1116]: Is the seq list redundant (i.e not needed for downstream code), or what's the reason for dropping the seq list in this PR?

**Review by mattssca — APPROVED** [review]
Nice edits, I've tried the updated function and it does what's advertised. No changes requested, see comments in PR for more info.

File: R/utilities.R (hunk 2 above)
Comment (kristenadaley) [reply R/utilities.R:1116]: I am keeping the list but just have to specify the maf (at the end), otherwise it can throw an error

**Pull request merged**